src/main/java/com/minecolonies/coremod/colony/managers/RaidManager.java

```diff
@@ -118,7 +118,7 @@
     /**
      * The dynamic difficulty of raids for this colony
      */
-    private int raidDifficulty = MIN_RAID_DIFFICULTY;
+    private int raidDifficulty = 5;
```

Comment:
why adjust the value instead of the constant?

Reply:
because I don't want to increase the minimum scaling, but the starting level

Reply:
Should we make it a START_RAID_DIFFICULTY then?

Reply:
no not used elsewhere

Reply:
I know it's not used elsewhere, but it's to not have it be a magic number but "explained"